### PR TITLE
remove dead/unused code.

### PR DESCRIPTION
**Pull request by james-lawrence**

### CODE REVIEW

**Comment by james-lawrence** [comment]
as an aside, outside of druid APIs the wayland code is fairly functional. mostly keyboard (repeating keys) and output selection remains.

File: druid-shell/src/backend/wayland/mod.rs [hunk 1]
```diff
@@ -22,7 +22,6 @@ mod events;
 pub mod keyboard;
 pub mod menu;
 pub mod pointers;
-pub mod screen;
```
Comment (Maan2003) [inline druid-shell/src/backend/wayland/mod.rs:25]: wait you can't just remove screen, druid shell depends on this and will not compile

Comment (james-lawrence) [reply druid-shell/src/backend/wayland/mod.rs:25]: except I did, and it does.... =)

Comment (james-lawrence) [reply druid-shell/src/backend/wayland/mod.rs:25]: ahh i lied. I was building on a different branch. will readd the method. =(

Comment (Maan2003) [reply druid-shell/src/backend/wayland/mod.rs:25]: I think it is fine to leave it as stub.

Comment (james-lawrence) [reply druid-shell/src/backend/wayland/mod.rs]: fixed up and pulled in 1 more small change that fixes an issue with the global id truly being a constant. (w/ a value of 1)

**Pull request merged**